Android/app/src/main/java/io/github/project_travel_mate/utilities/WorldClockActivity.java

```diff
@@ -2,44 +2,38 @@
 
 import android.content.Context;
 import android.content.Intent;
-import android.graphics.Color;
+import android.content.SharedPreferences;
 import android.os.Bundle;
+import android.preference.PreferenceManager;
 import android.support.annotation.Nullable;
 import android.support.v7.app.AppCompatActivity;
-import android.text.format.DateFormat;
-import android.util.Log;
 import android.view.MenuItem;
-import android.widget.ImageView;
+import android.view.View;
+import android.widget.AdapterView;
+import android.widget.ArrayAdapter;
+import android.widget.Spinner;
 import android.widget.TextClock;
-import android.widget.TextView;
+import com.tomerrosenfeld.customanalogclockview.CustomAnalogClock;
```

Comment:
What was the issue with earlier code?
We won't be adding a new dependency for world clock.

Reply:
In, the earlier code, the original library for the analog clock has issues (shows glitch effects and incorrect time) when the users choose another time zone. And also, the previous contributor for the world clock utility set the device local time as Kolkata's time. This becomes error when Users from different time zone use this utility.
So, I replaced with another analog library for showing the correct time.

Reply:
Dont use another library. use native code only to fix this issue